mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -136,6 +138,8 @@ CLASS net/minecraft/unmapped/C_astfners net/minecraft/entity/Entity
 		ARG 1 boundingBox
 	METHOD m_bqujtsnz isSubmergedInWater ()Z
 		COMMENT Returns whether this entity's hitbox is fully submerged in water.
+		COMMENT
+		COMMENT @return {@code true} if this entity's hitbox is fully submerged in water, otherwise {@code false}
```

Comment:
You should use a return tag here

Reply:
My issue with those tags is it doesn't allow precise return text and brief text.
So either we go detaield with the true if/otherwise false
or we just go with the whether formulation.